java/repositories.bzl

```diff
@@ -65,6 +65,12 @@ def java_tools_javac11_repos():
         ],
     )
 
+def local_jdk_repo():
+    local_java_repository(
+        name = "local_jdk",
+        build_file = Label("//toolchains:jdk.BUILD"),
```

Comment:
unrelated to this PR, but I recently had to apply this build file (well, the pre-moved one) to make a custom `local_java_repository` work with RBE. I was surprised this wasn't the default behavior.

Reply:
Do you mean `Label("//toolchains:jdk.BUILD")` should be the default value of `build_file`?

Reply:
While it's a breaking change, yes. Without that, I found that javabase inputs aren't passed to RBE, and instead they were found at the paths that `java_home` resolved to.

Reply:
Feel free to send a PR to the Bazel repo, we can then review it could be made in a way backwards compatible.